static/sass/_snapcraft_market.scss

```diff
@@ -6,19 +6,9 @@
   $black-overlay-light-color: rgba(0, 0, 0, .2);
   $white-overlay-strong-color: rgba(255, 255, 255, .9);
 
-  // Add extra space between groups of related items
-  .snapcraft-p-market__section {
-    margin-bottom: $sp-xx-large;
-  }
-
-  .market-form {
-    .row {
-      margin-bottom: $sp-medium;
-
-      & .row {
-        margin-bottom: 0;
-      }
-    }
+  // Override baseline alignment
+  .p-form--stacked .p-form__group {
+    align-items: flex-start;
```

Comment:
It should probably be a modifier (`.p-form__group--top`?) because it breaks alignment in some places:

<img width="622" alt="Screenshot 2019-08-16 at 15 18 07" src="https://user-images.githubusercontent.com/83575/63170247-294b9980-c039-11e9-95d0-0f9c22372018.png">